google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -2107,6 +2114,7 @@ public static Builder newBuilder() {
    *     granularity supported is 1 second, finer granularities will be truncated
    * @param unit time unit of the {@code duration} parameter
    * @param options optional URL signing options
+   *     {@code SignUrlOption.withHostName()} option to sign url with custom hostname.
```

Comment:
`option to set a custom hostname instead of using https://storage.googleapis.com.`

Reply:
resolved